stdlib/grpc/src/main/ballerina/src/grpc/annotation.bal

```diff
@@ -59,7 +59,7 @@ public annotation GrpcResourceConfig ResourceConfig on resource function;
 # + descMap - Service dependent descriptor map, which should be set at the compile time
 public type ServiceDescriptorData record {|
     string descriptor = "";
-    map<any> descMap = {};
+    map<anydata> descMap = {};
```

Comment:
@daneshk please help review if this change is okay.

Reply:
yes, it's okay